sqlglot/tokens.py

```diff
@@ -199,6 +199,7 @@ class TokenType(AutoName):
     AGGREGATEFUNCTION = auto()
     SIMPLEAGGREGATEFUNCTION = auto()
     TDIGEST = auto()
+    HYPERLOGLOG = auto()
```

Comment:
I don't think we need a new token, we already have `HLLSKETCH`.

Reply:
just to see if I'm on the right track:
I should remove HyperLogLog mentions I added to "expressions", "parser" and "tokens" files (since HyperLogLog has similar logic/handling as already defined HLLSKETCH), right? I should make use of HLLSketch in "presto" file?

I'm sorry if questions sounds silly, first time trying to contribute..

Reply:
basically add:
            exp.DataType.Type.HLLSKETCH: "HYPERLOGLOG",

Reply:
I've made changes, LMK if that's what you meant!